new-cdc/eventuate-local-java-cdc-connector-postgres-wal/src/main/java/io/eventuate/local/postgres/wal/PostgresWalClient.java

```diff
@@ -168,7 +157,22 @@ private void connectAndRun(Optional<BinlogFileOffset> binlogFileOffset)
 
         postgresWalBinlogEntryExtractor
```

Comment:
This code would probably be simpler of written using for/if statements

Reply:
I will fix